test/integration/esClient.js

```diff
@@ -7,12 +7,19 @@ const expect = chai.expect;
 const maxWaitTime = 1 * 60 * 1000;
 const indexName = 'profiles-test';
 const aliasName = 'profiles-alias';
+const orphanAlias = 'profiles-orphan';
+const index1 = `${orphanAlias}_20180117100000`;
+const index2 = `${orphanAlias}_20180117100010`;
+const index3 = `${orphanAlias}_20180117100020`;
 
 describe('Elasticsearch Client', function test() {
   this.timeout(maxWaitTime);
 
-  before((done) => {
-    esClient.waitForHealthyEs().then(done);
+  before(async () => {
```

Comment:
I wonder whether it is worth running the deletion of the indexes in an after block too. The reason would be that after the last test has run there will be `index3` still existing so if another test was added there might be some inconsistent or unexpected results if the person making the change didn't realise that `index3` would still exist.